frontend/src/modules/member/components/view/_aside/_aside-identities.vue

```diff
@@ -42,11 +42,16 @@
           target="_blank"
           rel="noopener noreferrer"
         >
-          <div class="flex gap-3 items-center">
-            <app-platform platform="email" />
-            <span class="text-gray-900 text-xs">
-              {{ email }}</span>
-          </div>
+          <el-tooltip
+            placement="top"
+            :content="email"
+          >
+            <div class="flex gap-3 items-center overflow-hidden">
```

Comment:
Can you add a margin to this container so that it has some space between the text and icon? Can be `mr-2`

Reply:
there's a `ml-2` in the icon class already.

Reply:
or not, seems like I removed before pushing :)